lib/spack/spack/cmd/create.py

```diff
@@ -164,6 +168,10 @@ def install(self, spec, prefix):
         scons('prefix={0}'.format(prefix))
         scons('install')""",
 
+    'bazel': """\
+        # FIXME: Add logic to build and install here.
+        bazel('build')""",
```

Comment:
> But if we do that, it should be for all build systems, not just Bazel. This PR introduces support for Bazel. Another PR could move build system configuration to the package.py files. 

Ok, I'll try to submit a PR that goes in that direction.

> But specifying everything in build_environment.py is a lot less duplicated code.

Why do you think so ? As I see it each line that now is in `build_environment` and takes care of a build-system will just be moved to the corresponding package.